lib/oneview-sdk/resource.rb

```diff
@@ -361,6 +361,12 @@ def recursive_like?(other, data = @data)
         return false unless data && data.respond_to?(:[])
         if val.is_a?(Hash)
           return false unless data.class == Hash && recursive_like?(val, data[key.to_s])
+        elsif val.is_a?(Array) && val.first.is_a?(Hash)
```

Comment:
I agree with @jsmartt . Just check the impact on other features but I think this will work fine.